cfgov/unprocessed/apps/youth-employment-success/js/util.js

```diff
@@ -3,6 +3,21 @@ let UNDEFINED;
 const REDUCER_RETURN_ERROR = 'Reducer must return a state object';
 const INVALID_ARG_ERROR = 'reducers argument must be an object, where each value is a reducer function';
 
+/**
+ * Helper method to generate an action creator
+ * @param {string} actionType The name of the action to be dispatched
+ * @returns {function} Curried action creator function that accepts a
+ *   data argument and returns an action
```

Comment:
Should it say "action object." ?

Reply:
I believe `action` and `action object` are essentially synonymous in this pattern